testing/test_runner.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+import context
+from roboplot.core.gpio import gpio_wrapper as gpio_wrapper
+
+
+class CustomTestRunner(unittest.TextTestRunner):
+    """ A test runner which behaves the same as the TextTestRunner, but which closes the emulator gui when all tests
+    have been run."""
+    def run(self, test):
+        result = super().run(test)
+        try:
+            gpio_wrapper.clean_up()
+        finally:
+            return result
```

Comment:
Well, I'm presuming that the TextTestRunner does all the catching and handling it needs to, so no need for a `try` around that (we don't have one when we use TextTestRunner directly!).
Presumably though there's no harm in putting it inside the `try` block if you want to though 😃 

However, if the clean-up fails for some reason (I don't have anything in particular in mind) then I presumed we'd still want the results of the test.